lib/native-package-installer/platform/ubuntu.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2013  Ruby-GNOME2 Project Team
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+class NativePackageInstaller
+  module Platform
+    class Ubuntu
```

Comment:
We should use `NativePackageInstaller::Platform::Debian` as the parent class instead of defining this class from scratch.
See also: `alt-linux.rb`

Reply:
3dea1a5